jax/extend/core/__init__.py

```diff
@@ -19,4 +19,15 @@
   array_types as array_types
 )
 
+from jax._src.core import (    
```

Comment:
Our lint check seems unhappy about trailing whitespace on this line.

Reply:
fixed...